src/test/java/org/killbill/billing/plugin/api/notification/TestPluginTenantConfigurableConfigurationHandler.java

```diff
@@ -81,6 +87,28 @@ public void testConfigurationForTenant() throws Exception {
         Assert.assertEquals(configurationHandler.getConfigurable(null), "DEFAULT");
     }
 
+    @Test(groups = "fast")
+    public void testCloseable() throws Exception {
+        final UUID configuredTenant = UUID.randomUUID();
+        final UUID nonConfiguredTenant = UUID.randomUUID();
+
+        // configure tenants
+        mockTenantKvsCloseable(configuredTenant, ImmutableList.<String>of("CloseableTest"), nonConfiguredTenant, ImmutableList.<String>of());
+        closableConfigurationHandler.configure(configuredTenant);
```

Comment:
Is this line really necessary -- the method getConfigurable will call `configure` [here](https://github.com/killbill/killbill-plugin-framework-java/blob/work-for-release-0.21.x/src/main/java/org/killbill/billing/plugin/api/notification/PluginTenantConfigurableConfigurationHandler.java#L66) ?

Reply:
- only for a [new tenant](https://github.com/killbill/killbill-plugin-framework-java/blob/5f1cbae25e8758990cc310e1bab3f0c9035b1d68/src/main/java/org/killbill/billing/plugin/api/notification/PluginTenantConfigurableConfigurationHandler.java#L62) (not in the `configuredTenants` set of the handler)
- `closableConfigurationHandler.configure(configuredTenant);` is the call that would be made when the framework receives a [config update](https://github.com/killbill/killbill-plugin-framework-java/blob/5f1cbae25e8758990cc310e1bab3f0c9035b1d68/src/main/java/org/killbill/billing/plugin/api/notification/PluginConfigurationEventHandler.java#L37)